Tasks/PublishPipelineArtifactV1/task.json

```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 0,
+        "Minor": 1,
         "Patch": 0
```

Comment:
Is this going to break existing YAML pipelines? The artifactType field is a required field now, I know it defaults to "pipeline" but does YAML break because the field is missing from existing pipelines?

Reply:
I'm setting required to false so it won't break the old pipelines but we will need to update yaml schema so that users can configure these new input fields in yaml.